changes/2841.feature.txt

```diff
@@ -0,0 +1 @@
+``toga.App`` now has a ``dark_mode`` read-only property, to read the user's OS-level dark mode preferences. Support is currently limited to Cocoa, anod other backends will use ``None`` to indicate that the dark mode preference cannot be detected.
```

Comment:
We don't need to fully document the feature in the changenote; we're just advertising that it exists. The API docs become the "full" documentation.
```suggestion
Apps can now interrogate whether they are in dark mode on some platforms.
```